.github/workflows/ci.yml

```diff
@@ -22,4 +22,7 @@ jobs:
         go-version: '1.22'
         cache: true
     - name: Test
-      run: go test -covermode atomic ./...
+      run: go test ./...
+    - name: Test Coverage
+      if: matrix.os == 'ubuntu-latest'
+      run: go test -covermode=atomic `go list ./... | grep -v -f .test_ignore.txt`
```

Comment:
@patzielinski and I discussed this and we explicitly decided to keep this separate from the main test step. This is so that we don't accidentally skip running tests on ubuntu because we forgot to update .test_ignore.txt or so. Even with test performance hits (#457), ubuntu is plenty fast that we can afford to effectively run tests twice.

Reply:
If we are ok with only having coverage for ubuntu, then this looks good to me.

Reply:
yeah, we have almost no OS specific code. The exception is testing for encrypted ssh keys on windows. I think we can safely just test on ubuntu.